x-pack/plugins/alerting/server/task_runner/action_scheduler/rule_action_helper.ts

```diff
@@ -129,3 +125,25 @@ export const getSummaryActionTimeBounds = (
 
   return { start: startDate.valueOf(), end: now.valueOf() };
 };
+
+interface LogNumberOfFilteredAlertsOpts {
+  logger: Logger;
+  numberOfAlerts: number;
+  numberOfSummarizedAlerts: number;
+  action: RuleAction;
+}
+export const logNumberOfFilteredAlerts = ({
```

Comment:
this was a private function in `ExecutionHandler` class. Moved to a reusable helper function

Reply:
Would be nice if there are some basic unit tests for this function.

Reply:
Updated in https://github.com/elastic/kibana/pull/186666/commits/86a902419d4bbe629ac94d25e6878dd7c5b07745